exonum/src/lib.rs

```diff
@@ -27,6 +27,8 @@ extern crate exonum_rocksdb as rocksdb;
 #[macro_use]
 extern crate exonum_profiler;
 #[macro_use]
+extern crate exonum_encoding as encoding;
```

Comment:
I think this might want to be `pub extern crate` for `$crate::encoding` in macros to work?